api/games/server/cron.js

```diff
@@ -39,21 +39,102 @@ Cron.add({
           return;
         }
         const botPlayers = Players.find(query);
-        const players = Players.find({ gameId }).fetch();
         const treatment = Treatments.findOne(game.treatmentId);
         const round = Rounds.findOne(stage.roundId);
-        game.treatment = treatment.factorsObject();
-        game.players = players;
-        game.rounds = Rounds.find({ gameId }).fetch();
-        game.stages = Stages.find({ gameId }).fetch();
+
+        let gameTreatment = null,
+          gamePlayers = null,
+          gameRounds = null,
+          gameStages = null;
+
+        Object.defineProperties(game, {
+          treatment: {
+            get() {
+              if (!gameTreatment) {
+                gameTreatment = treatment.factorsObject();
+              }
+
+              return gameTreatment;
+            }
+          },
+          players: {
+            get() {
+              if (!gamePlayers) {
+                gamePlayers = game.getPlayers();
+                gamePlayers.forEach(player => {
+                  let playerRound = null,
+                    playerStage = null;
+
+                  Object.defineProperties(player, {
+                    round: {
+                      get() {
+                        if (!playerRound) {
+                          playerRound = _.extend({}, round);
+                        }
+
+                        return playerRound;
+                      }
+                    },
+                    stage: {
+                      get() {
+                        if (!playerStage) {
+                          playerStage = _.extend({}, stage);
+                        }
+
+                        return playerStage;
+                      }
+                    }
+                  });
+
+                  augmentPlayerStageRound(
+                    player,
+                    player.stage,
+                    player.round,
+                    game
```

Comment:
It's the same here. Please verify everywhere you are not calling the lazy loaded data just after defining it.
But more importantly, looking all these changes, I see a lot of code duplication. I think you should be able to create functions to reuse code more (and thus limit maintenance cost and bug potential). A bit like the augment functions do. Let me know if this isn't clear, or you don't see how to reduce code duplication. 🙌